### PR TITLE
upd install script

### PR DESCRIPTION
**Pull request by smagolexandr**
* add node dependecies
* upd install script
* upd travis.yml

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,36 +1,12 @@
...
-esac
\ No newline at end of file
+echo "Installing project";
+
+composer install;
+./bin/console d:d:c;
+./bin/console d:s:c;
+./bin/console h:f:l -n;
+npm install
+bower install --allow-root
```
Comment (spolischook) [inline install.sh]: why you need ```--allow-root``` - this is bad practice you know?

File: install.sh [hunk 2]
```diff
@@ -1,36 +1,12 @@
...
-      ./bin/console c:c
-    ;;
-
-   "0" ) ;;
-esac
\ No newline at end of file
+echo "Installing project";
+
+composer install;
+./bin/console d:d:c;
```
Comment (spolischook) [inline install.sh]: This looks like a magic and usefull only for less typing, use full commands here.

Comment (smagolexandr) [inline install.sh]: fixed

File: install.sh (hunk 1 above)
Comment (smagolexandr) [inline install.sh]: fixed

**Pull request merged**